docs/pages/guides/using-eslint.mdx

```diff
@@ -1,13 +1,15 @@
 ---
-title: Use ESLint
-description: A guide on configuring ESLint to format Expo apps.
+title: Use ESLint and Prettier
+description: A guide on configuring ESLint and Prettier to format Expo apps.
 ---
 
+import { BoxLink } from '~/ui/components/BoxLink';
+import { CODE } from '~/ui/components/Text';
 import { Collapsible } from '~/ui/components/Collapsible';
-import { Terminal } from '~/ui/components/Snippet';
 import { FileTree } from '~/ui/components/FileTree';
-import { Tabs, Tab, TabsGroup } from '~/ui/components/Tabs';
 import { Step } from '~/ui/components/Step';
+import { Tabs, Tab, TabsGroup } from '~/ui/components/Tabs';
+import { Terminal } from '~/ui/components/Snippet';
 
 <TabsGroup>
 
```

Comment:
We have a short sentence about ESlint. We could also add a sentence on Prettier here, just after the eslint one and before `## Setup`? Something like:

> [Prettier](https://prettier.io/) is a code formatter which helps ensure that all outputted code conforms to a consistent style. It is often used in conjunction with ESlint

Reply:
Good point! Let me update the introduction.

Reply:
Updated the introduction of this guide:

<img width="848" alt="CleanShot 2023-10-23 at 16 17 57@2x" src="https://github.com/expo/expo/assets/10234615/76520bf3-77d4-4e81-b078-6c7055c926cc">

Also removed prettier's intro from the section, since, now we have that in the first paragraph:

<img width="889" alt="CleanShot 2023-10-23 at 16 18 01@2x" src="https://github.com/expo/expo/assets/10234615/4676e198-ba3e-49e4-91cd-f8060c9624fa">